skills/relay-dispatch/scripts/manifest/pr-number-stamp.js

```
@@ -0,0 +1,173 @@
+      ...freshRecord,
+      data: updatedData,
+    };
+  } finally {
+    try {
+      fs.closeSync(lockFd);
+    } catch {}
+    try {
+      fs.unlinkSync(lockPath);
+    } catch (error) {
+      if (error.code !== "ENOENT") {
+        throw error;
```

Biome (2.4.12)

[error] Unsafe usage of 'throw'.

(lint/correctness/noUnsafeFinally)